swagger/oauth.yaml

```diff
@@ -9,6 +9,11 @@ endpoints:
       operationId: request_access_token
       tags:
         - oauth2
+      description: |
+        To request an access token, the client obtains authorization from the
+        resource owner.  The authorization is expressed in the form of an
+        authorization grant, which the client uses to request the access
+        token. See https://tools.ietf.org/html/rfc6749#section-4 for more detail.
```

Comment:
Isn't that just the "password" authorization grant type? I thought this statement was generic enough to cover password + client credentials but I may be incorrect.